buildtest/schemas/spack.schema.json

```diff
@@ -118,6 +118,21 @@
             }
           }
         },
+        "deactivate": {
+          "additionalProperties": false,
+          "type": "object",
+          "description": "Deactivate a spack environment prior to activating it via ``spack env deactivate``",
+          "properties": {
+            "name": {
+              "type": "boolean",
+              "description": "Deactivate any active spack environment before activating the environment. If set to ``True`` we will run ``spack env deactivate``."
+            },
```

Comment:
okay we need this to be a boolean type not an object can you please remove the `properties` it just needs to be a boolean.

In the buildspec we have

```
deactivate: true
```

Right now your schema is setup to be something like this in yaml

```yaml
deactivate:
  name: true
  options: "--help"
```

Reply:
Okay. Thank you.
I have fixed it